include/seqan3/io/detail/misc_input.hpp

```diff
@@ -99,44 +99,57 @@ inline auto make_secondary_istream(std::basic_istream<char_t> & primary_stream,
     for (size_t i = 0 ; i < read_chars; ++i)
         primary_stream.unget();
 
-    std::string const extension = filename.extension().string();
+    std::string extension{};
+    if (filename.has_extension())
+        extension = filename.extension().string().substr(1);
 
     // set return value appropriately
     if (read_chars == magic_number.size() && contrib::_bgzfCheckHeader(magic_number.data())) // BGZF
```

Comment:
Is this now valid when Gzip is not available? I remember we had an issue on that.

Reply:
There was an issue with some functionality not available if Gzip was not found. I'll fix this in another PR this or next week